scripts/json_verify.py

```diff
@@ -5,9 +5,23 @@
 import glob
 import json
 import re
+import sys
+import traceback
+from collections import Counter
 from types import DictType, ListType, UnicodeType
 from urlparse import urlparse
 
+FINGERPRINTING_TAG = 'fingerprinting'
+CRYPTOMINING_TAG = 'cryptominer'
+SESSION_REPLAY_TAG = 'session-replay'
+PERFORMANCE_TAG = 'performance'
+ALL_TAGS = {
+    FINGERPRINTING_TAG,
+    CRYPTOMINING_TAG,
+    SESSION_REPLAY_TAG,
+    PERFORMANCE_TAG
+}
```

Comment:
Is there a reason to use `{` and `}` syntax making this a `set` rather than a list?

Reply:
Yes. Set inclusion is faster than list inclusion and I originally wrote this to check if the tag read was in the set. With lists/sets this small I don't think it will matter, but since we don't actually check set inclusion anymore, we can just convert this to a list.